apps/web/src/state/user/smartRouter.ts

```diff
@@ -9,7 +12,8 @@ const userUserSplitRouteAtom = atomWithStorageWithErrorCatch<boolean>('pcs:useSp
 const userUseXAtom = atomWithStorageWithErrorCatch<boolean | undefined>('pcs:useX', undefined)
 
 export function useUserXEnable() {
-  return useAtom(userUseXAtom)
+  const reset = useResetAtom(userUseXAtom)
```

Comment:
Want to double-check, is our atom resettable to be used in `useResetAtom`?

https://jotai.org/docs/utilities/resettable

Reply:
Okay, got it. Pretty nuanced thing lol. Thanks for the explanation, everything else lgtm 👍🏻